pkg/controllers/selection/controller.go

```diff
@@ -96,13 +100,29 @@ func (c *Controller) selectProvisioner(ctx context.Context, pod *v1.Pod) (errs e
 	if len(provisioners) == 0 {
 		return nil
 	}
+
+	// lookup the pod namespace for matching against the provisioner
+	var podNamespace v1.Namespace
```

Comment:
I'd leave things unqualified unless it's unclear
```suggestion
	var namespace v1.Namespace
```